src/lib/viewers/controls/color-picker/ColorPickerControl.scss

```diff
@@ -1,5 +1,11 @@
 @import '../styles';
 
+@mixin highlight-style {
```

Comment:
We've followed a convention of namespacing mixins with`ba-`

Reply:
Sounds good - I think you mean `bp-` instead?